Jenkinsfile

```
@@ -228,6 +230,24 @@ void stageAutomaticRelease() {
             registry.pushK8sYaml(targetOperatorResourceYaml, repositoryName, "k8s", "${controllerVersion}")
         }
 
+        stage('Push Helm chart to Harbor') {
+            new Docker(this)
+                .image("golang:${goVersion}")
+                .mountJenkinsUser()
+                .inside("--volume ${WORKSPACE}:/go/src/${project} -w /go/src/${project}")
+                        {
+                            make 'k8s-helm-package-release'
+
+                            Makefile makefile = new Makefile(this)
+                            String controllerVersion = makefile.getVersion()
+
+                            withCredentials([usernamePassword(credentialsId: 'harborhelmchartpush', usernameVariable: 'HARBOR_USERNAME', passwordVariable: 'HARBOR_PASSWORD')]) {
+                                sh ".bin/helm registry login ${registry} --username '${HARBOR_USERNAME}' --password '${HARBOR_PASSWORD}'"
+                                sh ".bin/helm push target/helm/${repositoryName}-${controllerVersion}.tgz oci://${registry}/${registry_namespace}/"
```

I think the directory of the packe is `target/make/k8s/helm`
```suggestion
                                sh ".bin/helm push target/make/k8s/helm/${repositoryName}-${controllerVersion}.tgz oci://${registry}/${registry_namespace}/"
```

---

I see where you coming from, but the main Makefile sets the Variable first to: 

`K8S_RESOURCE_TEMP_FOLDER ?= $(TARGET_DIR)`

And my smoketests are resulting in just target/helm/ as well. The Jenkins also worked with the current settings. But it is indeed confusing.